smetrics/src/main/scala/com/evolutiongaming/smetrics/MeasureDuration.scala

```diff
@@ -59,4 +59,33 @@ object MeasureDuration {
       val start = f(self.start.map(f.apply))
     }
   }
+
+  object implicits {
+    implicit class OpsMeasuredDuration[F[_], A](val fa: F[A]) extends AnyVal {
+      def measured(
+        handleF: FiniteDuration => F[Unit]
+      )(implicit F: Monad[F], measureDuration: MeasureDuration[F]): F[A] =
+        for {
+          measure <- measureDuration.start
+          result <- fa
+          duration <- measure
+          _ <- handleF(duration)
+        } yield result
```

Comment:
add empty line after this… please

Reply:
Can `measured` be implemented as a `measuredCase` special case? DRY and all that...

Reply:
Yeah, also then it would require `MonadError` instead of just `Monad`, so never mind